packages/angular_devkit/build_angular/test/browser/aot_spec_large.ts

```diff
@@ -40,30 +40,6 @@ describe('Browser Builder AOT', () => {
     await run.stop();
   });
 
-  it('shows warnings for component styles', async () => {
```

Comment:
Can you provide some context on why is this test (and the next one) removed?

Reply:
CSSnano doesn't output such warning for these.